Week1/insertValues.js

```diff
@@ -0,0 +1,52 @@
+const mysql = require('mysql');
+const connection = mysql.createConnection({
```

Comment:
Tip: This connection to DB is duplicated in several files. With this approach, if we decided to change the DB credentials, we'll need to change all of the files.

Instead, we could refactor the code so that the credentials are stored in only one file or at least in a config file that we read from in each of the other files.